app/views/govuk_publishing_components/components/_intervention.html.erb

```diff
@@ -0,0 +1,24 @@
+<%
+  local_assigns[:margin_bottom] ||= 8
+  local_assigns[:margin_bottom] = 8 if local_assigns[:margin_bottom] > 9
+
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+
+  css_classes = %w[gem-c-intervention]
+  css_classes << (shared_helper.get_margin_bottom)
```

Comment:
If this component uses the shared helper to provide an option for different bottom margins, that should be documented in the yml file.

Reply:
Left over from copied element. Removing.